src/Containers/LandingPage/LandingPage.js

```diff
@@ -0,0 +1,62 @@
+import Navbar from './NavBar/navbar/navbar'
+import MobileNavBar from './NavBar/MobileNavBar/MobileNavBar'
+import HomePage from '../HomePage/HomePage'
+import {Redirect, Route,Switch,useLocation} from 'react-router-dom'
+import Catagories from '../Catagories/Catagories'
+import {CatagoriesContext} from '../../Store/CatagoriesReducer'
+import {AuthContext} from '../../Store/AuthReducer'
+import PlaylistPage from '../PlaylistPage/PlaylistPage'
+import {useEffect,useContext} from 'react'
+import VideoPlayer from '../Catagories/VideoPlayer/VideoPlayer'
+import HistoryPage from '../History/History'
+import LoginPage from '../LoginPage/LoginPage'
+import Spinner from '../../UI/Spinner/Spinner'
```

Comment:
U can improve the file folder naming here... and group all your exports from a folder in an index file

Reply:
Hey Danish, in my directory structure it doesn't make much difference if I an index.js file since all the components used by the page will be in the same directory.